Styling/README.md

```diff
@@ -501,6 +502,18 @@ Returns the cosine value of the number (in radians).
 }
 ```
 
+#### sqrt 
+
+`sqrt(value : Number) : Number`
+
+Returns the square root value of the number.
+
+```json
+{
+    "show" : "sqrt(${temperature}) > 0.0"
```

Comment:
Perhaps use a more realistic example like use `sqrt` to create a color ramp using the `"color"` property instead of `"show"`.